### PR TITLE
Release 2020 04 13

### PR DESCRIPTION
**Pull request by dpopp07**
Code regenerated to incorporate latest service updates.

Most notably, parameters were added to the HTTP and WebSocket `recognize` methods for Speech to Text.

### CODE REVIEW

File: visual-recognition/v4.ts [hunk 1]
```diff
@@ -1914,7 +1914,7 @@ namespace VisualRecognitionV4 {
      */
     object: string;
     /** Number of bounding boxes in the collection with the updated object name. */
-    count: number;
+    count?: number;
```
Comment (mamoonraja) [inline visual-recognition/v4.ts:1917]: any idea why this changed?

File: lib/recognize-stream.ts [hunk 2]
```diff
@@ -177,6 +177,8 @@ class RecognizeStream extends Duplex {
       'redaction',
       'end_of_phrase_silence_time',
       'split_transcript_at_phrase_end',
+      'speech_detector_sensitivity',
```
Comment (mamoonraja) [inline lib/recognize-stream.ts:180]: we need to add these here: https://github.com/watson-developer-cloud/node-sdk/blob/35fb2a5c9692c6e8a67f9de969269a8d37eb2ebb/lib/recognize-stream.ts#L105

File: visual-recognition/v4.ts (hunk 1 above)
Comment (dpopp07) [reply visual-recognition/v4.ts:1917]: It changed in the API Definition

File: lib/recognize-stream.ts (hunk 2 above)
Comment (dpopp07) [reply lib/recognize-stream.ts:180]: Good catch, thanks

File: visual-recognition/v4.ts (hunk 1 above)
Comment (mamoonraja) [inline visual-recognition/v4.ts:1917]: Confirming if this will change after generating with new `API def`?

File: assistant/v2.ts [hunk 3]
```diff
@@ -519,8 +519,20 @@ namespace AssistantV2 {
   export interface MessageContextSkill {
     /** Arbitrary variables that can be read and written by a particular skill. */
     user_defined?: JsonObject;
-    /** For internal use only. */
-    system?: JsonObject;
+    /** System context data used by the skill. */
+    system?: MessageContextSkillSystem;
```
Comment (mamoonraja) [inline assistant/v2.ts:523]: Does generating with the new API def tag still generate this change?

Comment (dpopp07) [reply assistant/v2.ts:523]: Re-generated the SDK, this change should not be present anymore

**Pull request merged**